src/Utils/commonFunctions.js

```
@@ -1,78 +1,158 @@
+<<<<<<< HEAD
+export function getPathName() {
+  const path = window.location.href;
+  const pathArray = path.split("/");
+  const pathname = pathArray[pathArray.length - 1];
+  return pathname;
+}
+
+export function convertTimeFormat(startDate, endDate) {
+  // console.log(startDate,endDate)
+  function formatTime(time) {
+    let [hours, minutes] = time?.split(":").map(Number);
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)